funding/manager.go

```diff
@@ -553,6 +553,19 @@ const (
 	addedToRouterGraph
 )
 
+func (c channelOpeningState) String() string {
```

Comment:
can probably drop this as well

Reply:
I think the string can be helpful in reading the logs?

Reply:
What was it before? If they are now stringified I agree, but I think it could be in a separate `multi` pull for log-changes

Reply:
Made its own commit instead. Let me know what you think.